assets/app/view/game/merge.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'view/game/actionable'
+require 'view/game/undo_and_pass'
+
+module View
+  module Game
+    class Merge < Snabberb::Component
+      include Actionable
+
+      def render
+        corporation = @game.mergable_candidates.first
+        merge = lambda do
+          process_action(Engine::Action::Merge.new(
+            @game.merge_target,
+            corporation: corporation,
+          ))
+        end
+
+        children = [h(:div, "Merge candidates: #{@game.mergable_candidates.map(&:name)}")]
+        children << h(:div, "To make the decision: #{@game.merge_decider.player.name}")
```

Comment:
This is a bit of a different flow than normal, but we are hitting issues where the 'active' player is not one that's normally listed on the round including in 1817.

Maybe we should have a standard way of showing that the normal turn sequence flow has been interrupted and who the interrupting entity is. But still showing the normal turn sequence as before.

@tobymao this would be nice to include on 1817 as the acquisiton is player based, but when your gaining/paying off loans it's corporation based. Would also help on the cash crisis as well, as it will get confusing whose turn it is.

Reply:
The current entity should be the 'merge_decider' anyway. So that the email gets sent to the right user.

Reply:
In g_18_mex/merge.rb, active_entities is [@game.merge_decider].
Normally, the merge_decider in 18MEX would be the president of the corporation to be merged into NdM. But there is one exception - if there is two(+) eligible corporations, that has not floated, then NdM will be the merge decider. It is probably a rare case.
If there is just one non-floated to choose from NdM is not asked but instead it is auto merged.

But NdM might have to select among multiple tokens placed, and that will also interrupt normal flow, before returning to Buy Trains step.

Reply:
Merging up to the latest should add the current entity to the list, removing the need for the "To make the decision" part

Reply:
Have removed the "To make the decision" part, moved the "Merge candidates" to the right of the Merge button. Rebased versus master.

Have not uploaded new screen shots as the changes are very minor.